linter/lintmain/internal/check/check_test.go

```diff
@@ -0,0 +1,39 @@
+package check
+
+import (
+	"testing"
+)
+
+func TestShortenLocation(t *testing.T) {
+	testGopath := "/home/queen/go/"
+	testGoroot := "/usr/lib/go/"
+	tests := []struct {
+		wd    string
+		input string
+		out   string
+	}{
+		{"", "/home/queen/go/file.go", "$GOPATH/file.go"},
```

Comment:
Only unix as I understood correctly?

Reply:
Oh, it should probably work for Windows if paths contain proper slashes. We're using HasPrefix, so it should match. We could probably add Windows-style paths here to test it...